apps/frontend/src/modules/users/libs/default-values/sign-up.default-values.ts

```diff
@@ -1,11 +1,11 @@
 import { type UserSignUpRequestDto } from "~/modules/users/users.js";
 
-const DEFAULT_SIGN_UP_PAYLOAD: UserSignUpRequestDto = {
+const DEFAULT_SIGN_UP_VALUES: UserSignUpRequestDto = {
```

Comment:
```suggestion
const USER_SIGN_UP_DEFAULT_VALUES: UserSignUpRequestDto = {
```
Also change file name to `user-sign-up-default-values.constant.ts`. Also change the folder name to `constants`